scipy/ndimage/filters.py

```diff
@@ -274,6 +305,9 @@ def gaussian_filter(input, sigma, order=0, output=None,
     truncate : float
         Truncate the filter at this many standard deviations.
         Default is 4.0.
+    weights : ndarray
```

Comment:
I see your point. I guess the advantage of this approach is not having to manually call `correlate1d` for each axis of the array manually, but that's hardly world ending.